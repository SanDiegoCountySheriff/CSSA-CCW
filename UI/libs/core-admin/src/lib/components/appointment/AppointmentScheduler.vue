<template>
  <div>
    {{ $t(' Appointment Scheduler I am under construction') }}
  </div>
</template>

<script setup lang="ts"></script>

<style lang="scss" scoped></style>
