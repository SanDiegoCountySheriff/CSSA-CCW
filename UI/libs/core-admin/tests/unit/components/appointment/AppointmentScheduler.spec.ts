/* eslint-disable @typescript-eslint/ban-ts-comment */
import AppointmentScheduler from '@core-admin/components/appointment/AppointmentScheduler.vue';
import Vuetify from 'vuetify';
import { createLocalVue, mount } from '@vue/test-utils';
import { createTestingPinia } from '@pinia/testing';

const localVue = createLocalVue();
const pinia = createTestingPinia();
const tMock = {
  $t: text => text,
};

describe('AppointmentScheduler', () => {
  let vuetify;
  let wrapper;

  beforeEach(() => {
    vuetify = new Vuetify();
    //@ts-ignore
    wrapper = mount(AppointmentScheduler, {
      localVue,
      vuetify,
      pinia,
      mocks: tMock,
    });
  });
  afterEach(() => {
    wrapper.destroy();
  });

  it('should match the snapshot', async () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
