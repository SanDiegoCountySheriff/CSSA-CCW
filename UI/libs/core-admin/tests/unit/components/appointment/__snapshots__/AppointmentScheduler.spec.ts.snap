// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppointmentScheduler should match the snapshot 1`] = `
<div>
  Appointment Scheduler I am under construction
</div>
`;
